### PR TITLE
Mesh 3 - min size criteria for edges, facets and cells

### PR DESCRIPTION
**Pull request by janetournois**
## Summary of Changes

This PR replaces PR #7315 and #7116 to avoid conflicts between them

## Release Management

* Affected package(s): Mesh_3
* License and copyright ownership: unchanged

### CODE REVIEW

File: Installation/CHANGES.md
Comment (lrineau) [inline Installation/CHANGES.md:1]: Warning: there are two sections in changes for 5.6, that should be merged:
- Tetrahedral Mesh Generation
- 3D Mesh Generation

**Comment by sloriot** [comment]
`test_min_edge_length` failed to compile in CGAL-5.6-Ic-256

**Comment by efifogel** [comment]
Yep; The polyline traits had to be treated as well.
Now, it is possible to draw arrangements induced by polylines as well.
Please test again...
   ____  _        ____             _
  /_____/_) o    /__________  __  //
 (____ (   (    (    (_/ (_/-(-'_(/
                         _/

**Comment by janetournois** [comment]
> `test_min_edge_length` failed to compile in CGAL-5.6-Ic-236

this PR is in conflict with PR #7307 which moves `Sizing_field_with_aabb_tree.h` to another location. So testing them together does not compile...
We should wait for one the two PR to be merged before we can merge the second one

**Comment by efifogel** [comment]
ops, sorry, wrong destination...
   ____  _        ____             _
  /_____/_) o    /__________  __  //
 (____ (   (    (    (_/ (_/-(-'_(/
                         _/

**Comment by lrineau** [comment]
> > `test_min_edge_length` failed to compile in CGAL-5.6-Ic-236
> 
> this PR is in conflict with PR #7307 which moves `Sizing_field_with_aabb_tree.h` to another location. So testing them together does not compile... We should wait for one the two PR to be merged before we can merge the second one

#7307 is now merged in `master`.

**Comment by sloriot** [comment]
Successfully tested in CGAL-5.6-Ic-247

**Pull request merged**